tests/test_rpms.py

```diff
@@ -658,6 +645,8 @@ def test_push_when_already_pushed(config_file, lookasidedir, default_config, cap
     git_url = default_config['git_push_url'] % {'package':'rcm-repoquery'}
     cmd = ['git', 'tag']
     out = check_output(cmd, cwd=git_url)
+    if not isinstance(out, str) and isinstance(out, bytes):
```

Comment:
Again it should be possible to instead add universal_newlines=True.

Reply:
Fixed.